actors.md

```diff
@@ -1,365 +1,14 @@
-# Getting Started
-snactor is a library written in Python to load, verify and execute an actor.
-An actor consists of a piece of code that can be executed and an yaml file that
-contains metadata about it, like: how it will be executed, messages it should
-receive as input, messages it will produce as output and so on. At this document
-we will describe how to write and execute a simple actor.
+# What is actor?
+An actor in terms of the Leapp project is a step that is executed within a workflow. Actors define what kind of data they expect and what they produce. Actors also provide a list of tags, with which actors mark their use cases.
 
+Actors scan the system and produce the information found as messages. Other actors consume those messages to make decisions or process the data to produce new messages. Some actors might apply changes to the system based on the information gathered earlier.
 
-## Defining what the actor will do
+If you want to know how to write actors, check [tutorial](http://leapp.readthedocs.io/en/latest/tutorials.html).
```

Comment:
Thinking about this, I believe it should be after the "# Why do I need to write actors?" section